www/src/data/ecosystem/featured-items.yaml

```diff
@@ -0,0 +1,12 @@
+starters:
```

Comment:
I don't think this is going to be enough -- the repo names can have conflicts. See similar issue here in the starter library: #9249

Reply:
I will use full slug for the starters

Reply:
It looks like this still needs to change?

Reply:
Why, the PR https://github.com/gatsbyjs/gatsby/issues/9249 is not merged yet, so if I would add the owners to the slug, I will not work. Am I wrong?

Reply:
@amandeepmittal Of course when the PR will be merged we have to update the slugs in the `featured-items.yaml`

Reply:
Oh, correct. I'll create a new issue dependent on that one getting merged.